hbase-hadoop2-compat/src/main/java/org/apache/hadoop/hbase/replication/regionserver/MetricsReplicationSinkSourceImpl.java

```diff
@@ -49,6 +51,11 @@ public MetricsReplicationSinkSourceImpl(MetricsReplicationSourceImpl rms) {
     opsCounter.incr(batchsize);
   }
 
+  @Override public void incrFailedBatches(){
+//    failedBatchesCounter.incr(batches);
+    failedBatchesCounter.incr();
```

Comment:
The `appliedBatches` always increments by `1` so I figured it would be better to just use the default `incr` method here.

Reply:
Sounds good, can you remove the commented out one?